rollingpin/args.py

```diff
@@ -81,6 +81,15 @@ def _add_iteration_arguments(config, parser):
         dest="pause_after",
     )
 
+    iteration_group.add_argument(
+        "--timeout",
+        default=None,
```

Comment:
Ah, I see now.  I'll do it that way to keep it consistent.  I don't think I've seen a CLI that output actual config values in its `--help` before though!  Makes sense.